src/Loading/Plugins/babylon.babylonFileLoader.ts

```diff
@@ -73,6 +73,7 @@
                     var parsedMaterial = parsedData.materials[index];
                     var mat = Material.Parse(parsedMaterial, scene, rootUrl);
                     container.materials.push(mat);
+                    container.scene.onNewMaterialAddedObservable.notifyObservers(mat);

```

Comment:
I think this one should be removed as the material is not in the scene technically

Reply:
done : 52520ba30c977914c2d18bd23b1f7ffb9df2e93a